phpstan-common.neon.dist

```diff
@@ -35,6 +37,11 @@ parameters:
             paths:
                 - tests/Doctrine/Migrations/Tests/Generator/ClassNameGeneratorTest.php
 
+        # TODO: Be more precise about class-strings
```

Comment:
I tried to fix this one error, but it's really a rabbit hole. Someone (maybe me) could work on a fix this on a dedicated PR. For now, it should be fine to ignore the error.